tests/tensor/test_math_opt.py

```diff
@@ -2888,46 +2887,27 @@ def test_local_log_erfc(self):
         mode_fusion.check_isfinite = False
 
         f = function([x], log(erfc(x)), mode=mode)
-        assert len(f.maker.fgraph.apply_nodes) == 23, len(f.maker.fgraph.apply_nodes)
+        assert len(f.maker.fgraph.apply_nodes) == 22
```

Comment:
One less node from the extra rewrite phase. These number of nodes tests seem a bit too brittle however...